style.css

```diff
@@ -0,0 +1,16 @@
+*,:after,:before{
+    box-sizing: border-box;
+    margin:0px;
+    padding:0px;
+}
+
+body{
+    display: flex;
+    justify-content: center;
+    align-items:center;
+    flex-direction: column;
+    background-color:green;
+}
+label{
+    font-weight: bolder;
+}
```

Comment:
- [optional] Please take note of this for knowledge sake, it is not advisable to use IDs as CSS selectors because if another element in the page uses the same/similar style, you would have to write the same CSS again. Even if you don't have more than one element with that style right now, it might come later. Hence it is always advisable to use class. 💁‍♂️